app/modules/parsing/graph_construction/parsing_helper.py

```
@@ -1884,8 +1913,29 @@ async def _clone_to_repo_manager(
+                    import traceback
+                    await email_helper.send_parsing_failure_alert(
+                        repo_name=repo_name,
+                        branch_name=ref,
+                        error_message="Worktree creation failed after successful bare repo clone",
+                        auth_method="environment",
+                        failure_type="worktree_creation",
+                        user_id=user_id,
+                        project_id=project_id,
+                        stack_trace=traceback.format_exc(),
+                    )
+                except Exception as email_err:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)